src/components/__tests__/StatsInnerList-test.tsx

```diff
@@ -2,22 +2,24 @@ import { expect } from "chai";
 import * as Sinon from "sinon";
 import * as Enzyme from "enzyme";
 import * as React from "react";
-import { testLibrary1, testLibrary2, modifyLibrary } from "./TestUtils";
+import { testLibrary1, testLibrary2, modifyLibrary, validate } from "./TestUtils";
 import StatsInnerList from "../StatsInnerList";
 
 describe("StatsInnerList", () => {
   let wrapper;
   let data;
   let productionLibrary1 = modifyLibrary(testLibrary1, { "name": "Production Library 1", "registry_stage": "production" });
+  productionLibrary1 = validate(productionLibrary1, productionLibrary1.basic_info.timestamp);
```

Comment:
This didn't make sense to me at first but it does after checking out the `validate` function. Add a comment here though to make this process easier to understand.